docs/filter-props.md

```diff
@@ -2,5 +2,264 @@
 id: filter-props
 title: Column Filter Props
 ---
+`react-bootstrap-table2` separate the filter core code base to [react-bootstrap-table2-filter](https://www.npmjs.com/package/react-bootstrap-table2-filter). The following are guideline about how to use and the details of props of [filterFactory](#filterfactory-props) and [filters](#filters-props). For more information, please take refer to the samples as [link](https://react-bootstrap-table.github.io/react-bootstrap-table2/storybook/index.html?selectedKind=Column%20Filter&selectedStory=Text%20Filter&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel) here.
 
-**No Any Available Props Yet**
\ No newline at end of file
+## Content Table
+
+* [Getting Started](#getting-started)
+* [How to use](#how-to-use)
+* [Props of FilterFactory](#props-of-filterfactory)
+* [Props of Filters](#props-of-filters)
+   * [textFilter](#1-textfilter)
+   * [selectFilter](#2-selectFilter)
+   * [numberFilter](#3-numberFilter)
+* [Comparator](#comparator)
+
+## Getting Started
+```
+$ npm install react-bootstrap-table2-filter --save
+```
+After installing `react-bootstrap-table2-filter`, you can configure `filter` on table as following instruction.
+
+## How to use
+You should apply following **2** to enable `filter` functionality for `react-bootstrap-table2`.
+* `filterFactory`
+* `filters` (**3** types support)
+   * textFilter
+   * selectFilter
+   * numberFilter
+
+```js
+import BootstrapTable from 'react-bootstrap-table-next';
+import filterFactory, { textFilter } from 'react-bootstrap-table2-filter';
+
+const columns = [{
+  dataField: 'id',
+  text: 'Product ID'
+}, {
+  dataField: 'name',
+  text: 'Product Name'
+}, {
+  dataField: 'price',
+  text: 'Product Price',
+  filter: textFilter()
+}];
+
+<BootstrapTable keyField='id' data={ products } columns={ columns } filter={ filterFactory() } />
+```
+
+## Props of FilterFactory
+**No Any Available Props Yet**
+
+## Props of Filters
+
+## (1). textFilter
```

Comment:
I think it's just little ugly on web page, please refine it. Other than, good to go!!